Export/AbstractExport.php

```diff
@@ -287,9 +287,9 @@ protected function formatDate(\DateTime $date, $format = 'short')
 
         switch ($format) {
             case 'short':
-                return $date->format('d-m-Y');
+                return $date->format('Y-m-d');
```

Comment:
Is this used in other exports?

Reply:
The formats should not be changed now (but I really wish that I'd initially gone with `Y-m-d`). I'll revert this change.